cds_migrator_kit/videos/weblecture_migration/load/helpers.py

```
@@ -0,0 +1,246 @@
+    celery_chain(task_signature).apply()
+    
+
+def copy_frames(payload, frame_paths):
+    """Load frames for the master video file without running the celery task.
+
+    payload (dict):
+        - version_id (str): The version ID of the main video file.
+        - key (str): The file name of the main video file.
+        - bucket_id (str): The bucket id of the record.
+        - deposit_id (str): The deposit id of the record.
+        - flow_id (str): The flow id.
+    frame_paths (list): A list of file paths of the frames.
+    """
+    try:
+        version_id = payload["version_id"]
+        object_version = as_object_version(version_id)
+
+        frames_task, kwargs = AVCFlowCeleryTasks.create_task(
+            ExtractFramesTask, payload, delete_copied=False
+        )
+        frames_task.flow_id = payload.get("flow_id")
+        # FramesTask Metadata
+        flow_task_metadata = frames_task.get_or_create_flow_task()
+        flow_task_metadata.status = FlowTaskStatus.STARTED
+
+        # Calculate time positions
+        options = frames_task._time_position(
+            duration=object_version.get_tags()["duration"],
+            frames_start=0, 
+            frames_end=90,
+            frames_gap=10,
+        )
+        # Create ObjectVersions with frames using ExtractFramesTask._create_frames
+        frames = frames_task._create_frames(
+            frame_paths,
+            object_version,
+            options.get("start_time"),
+            options.get("time_step"),
+        )
+        # Temp folder to create gif file
+        output_folder = tempfile.mkdtemp()
+        frames_task._create_gif(
+            bucket=str(object_version.bucket.id),
+            frames=frames,
+            output_dir=output_folder,
+            master_id=version_id,
+        )
+        flow_task_metadata.status = FlowTaskStatus.SUCCESS
+
+        # Cleanup
+        shutil.rmtree(output_folder)
+    except Exception:
```

the catch all exception block is not the best practice, if possible you should target specific exceptions (see the comment below)